optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -124,21 +339,26 @@ every computer whose processes consume more power than is available.
     }
 ----
 
-image::ConstraintStreams/constraintStreamGroupBy.png[align="center"]
+[NOTE]
+====
+Information may be lost during grouping.
+In the example above, `filter()` and all subsequent operations no longer have direct access to the original
```

Comment:
In the previous example...

We try to avoid using directional words in documents.